frontend/src/App.js

```diff
@@ -2,209 +2,229 @@ import { CssBaseline, ThemeProvider } from "@mui/material";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import { ColorModeContext, useMode } from "./theme";
 import { React, useState, useEffect } from "react";
-import LandingPage from './pages/LandingPage.js';
+import LandingPage from "./pages/LandingPage.js";
 import Topmenu from "./scenes/global/Topmenu";
 import Dashboard from "./scenes/dashboard";
 import Version from "./scenes/version";
 import AWS_Access from "./scenes/access";
 import Sidemenu from "./scenes/global/Sidemenu";
-import Webhooks from './scenes/webhooks';
-import Incident from './scenes/incident';
-import IncidentHistory from './scenes/incident_history';
-import Geolocate from './scenes/geolocate';
-import Faq from './scenes/faq';
-import Logout from './scenes/logout';
+import Webhooks from "./scenes/webhooks";
+import Incident from "./scenes/incident";
+import IncidentHistory from "./scenes/incident_history";
+import Geolocate from "./scenes/geolocate";
+import Faq from "./scenes/faq";
+import Logout from "./scenes/logout";
 
 /**
  * The main component of the application.
  */
 function App() {
+  // Get the current theme and color mode
+  const [theme, colorMode] = useMode();
 
-    // Get the current theme and color mode
-    const [theme, colorMode] = useMode();
+  // Set the initial state of the Sidemenu
+  const [isSidemenu, setIsSidemenu] = useState(true);
 
-    // Set the initial state of the Sidemenu
-    const [isSidemenu, setIsSidemenu] = useState(true);
+  // Set the state of whether a user is authenticated or not. Initially, this is set to null.
+  const [isAuthenticated, setIsAuthenticated] = useState(null);
 
-    // Set the state of whether a user is authenticated or not. Initially, this is set to null. 
-    const[isAuthenticated, setIsAuthenticated] = useState(null);
+  /**
+   * Custom hook to get user data from the server.
+   * @returns {boolean} Whether the user is authenticated or not.
+   */
+  const useUserData = () => {
+    const [userData, setUserData] = useState(null);
 
-    /**
-     * Custom hook to get user data from the server.
-     * @returns {boolean} Whether the user is authenticated or not.
-     */
-    const useUserData = () => {
-      const [userData, setUserData] = useState(null);
+    useEffect(() => {
+      const isDevelopment = process.env.NODE_ENV === "development";
+      const userUrl = isDevelopment ? "http://127.0.0.1:8000/auth/me" : "/auth/me";
+      // Make a GET request to the "/user" endpoint
```

Comment:
Wouldn't that be /me endpoint now?

Reply:
The auth module uses the prefix /auth and the endpoint is /me so the full endpoint is now `/auth/me`

Reply:
Oh absolutely! I didn't modify the inline comments, just the functions themselves